api-simple/next-steps.md

```diff
@@ -0,0 +1,67 @@
+# Next Steps
+
+## Table of Contents
+1. [Next Steps](#next-steps)
+2. [What was added](#what-was-added)
+3. [Billing](#billing)
+4. [Troubleshooting](#troubleshooting)
+
+## Next Steps
+### Define environment variables for running services
+
+Modify or add environment variables to configure the running application. Environment variables can be configured by modifying the `env` node in the following files:
+
+- [app/api-simple.bicep](./infra/app/api-simple.bicep)
+
+To define a secret as an environment variable, the secret can first be stored in KeyVault.
+
+### Provision infrastructure and deploy application code
+
+Run `azd up` to provision your infrastructure and deploy to Azure in one step (or run `azd provision` then `azd deploy` to accomplish the tasks separately). Visit the service endpoints from the command outut to see your application up-and-running!
+
+Note: To troubleshoot any issues, see [troubleshooting](#troubleshooting).
+
+## What was added
+
+To describe the infrastructure and application, `azure.yaml` along with Infrastructure as Code files using Bicep were added with the following directory structure:
+
+```yaml
+- azure.yaml     # azd project configuration
+- infra/         # Infrastructure as Code (bicep) files
+  - main.bicep   # main deployment module
+  - app/         # Application resource modules
+  - shared/      # Shared resource modules
+  - modules/     # Library modules
+```
+
+Each bicep file declares resources to be provisioned. The resources are provisioned when running `azd up` or `azd provision`.
+
+- [app/api-simple.bicep](./infra/app/api-simple.bicep) - Azure Container Apps resources to host the 'api-simple' service.
+- [shared/keyvault.bicep](./infra/shared/keyvault.bicep) - Azure KeyVault to store secrets.
+- [shared/monitoring.bicep](./infra/shared/monitoring.bicep) - Azure Log Analytics workspace and Application Insights to log and store instrumentation logs.
+- [shared/registry.bicep](./infra/shared/registry.bicep) - Azure Container Registry to store docker images.
+
+More information about [Bicep](https://aka.ms/bicep).
+
+If your project does not contain a Dockerfile, we will use [Buildpacks](https://buildpacks.io/) to create an image for the services in `azure.yaml` and get your containerized app onto Azure. 
+
+## Billing
+
+Visit the *Cost Management + Billing* page in Azure Portal to track current spend. For more information about how you're billed, and how you can monitor the costs incurred in your Azure subscriptions, visit [billing overview](https://learn.microsoft.com/en-us/azure/developer/intro/azure-developer-billing).
+
+## Troubleshooting
```

Comment:
@weikanglim @gkulin Do we also want to mention some common issues like
- `build` in azure.yaml pointing to the wrong folder (need to figure out what the convention is for the framework you're using, e.g. use your package.json build script and see what folder is generated, that's that's supposed to go here)
- @weikanglim  - not sure how we want to handle the DATABASE_URL problem. I think as a first step we might want to add something about setting a secret ref and then consuming it in the container app env variables?

Reply:
The `build` pointing to wrong folder "dissappears" when we switch over to the oryx builders and instead we'll deal with another set of issues™️.

For `DATABASE_URL`, another option is that you can build the string inside the python app by formatting everything. I've tweaked the secrets section slightly to explain this..